atomics/T1016/T1016.yaml

```diff
@@ -50,10 +50,11 @@ atomic_tests:
   executor:
     name: sh
     elevation_required: false
-    command: |
-      arp -a
-      netstat -ant | awk '{print $NF}' | grep -v '[a-z]' | sort | uniq -c
-      ifconfig
+    command: | # Check if the binary exists before calling it. Some conditions in the if statements are redundant to catch possible variations of "type" output.
+      if [ -x "$(command -v arp)" ]; then arp -a; else echo "arp is missing from the machine. skipping..."; fi;
+      if [ -x "$(command -v ifconfig)" ]; then ifconfig; else echo "ifconfig is missing from the machine. skipping..."; fi;
+      if [ -x "$(command -v ip)" ]; then ip addr; else echo "ip is missing from the machine. skipping..."; fi;
+      if [ -x "$(command -v netstat)" ]; then netstat -ant | awk '{print $NF}' | grep -v '[a-z]' | sort | uniq -c; else echo "netstat is missing from the machine. skipping..."; fi;
```

Comment:
I decided not to use dependencies in some cases because the commands aren't required to run the test. The technique above is an example. If we run the test on a recent distro (e.g. Ubuntu 19.10) we use `ip` which is pre-installed, but not `ifconfig` and the network information extracted are comparable between the two tools and we we don't need to install `ifconfig`. On the other end if we run the test on macOS we don't have, and don't need `ip`, but we use `ifconfig` which is shipped with macOS.

Reply:
Makes sense, thank you.